### ISSUE TITLE
ROI/ROAS Priors

### ISSUE THREAD

**Issue by louismagowan** [pymc-labs/pymc-marketing#905]
Hello!

Would it be possible to get ROI Priors added into pymc-marketing? As like in this [Google paper](https://research.google/pubs/media-mix-model-calibration-with-bayesian-priors/)

For example, they are available in [Meridian](https://developers.google.com/meridian/docs/basics/roi-priors).

I also saw that [Juan has done a PyMC implementation](https://juanitorduz.github.io/mmm_roas/) already for them.

I think it would be a super nice feature, and would greatly reduce the headaches that in-house MMMers encounter when trying to deliver models that align with stakeholders prior beliefs / expectations.

I have a nice idea of how they could be leveraged in the Streamlit app as well - and additional page could be added to the app which is dedicated to eliciting priors from stakeholders. In other words, I'm picturing that the app could be extended to function as a tool that MMMers use to gather priors from their stakeholders (which in my experience can be a very painful process otherwise).

**Comment by juanitorduz** [comment]
Yes, this would be a great addition! I am also planning to see if we can use https://www.pymc-marketing.io/en/stable/notebooks/mmm/mmm_lift_test.html to get similar results on the simulated dataset I generated.

**Comment by juanitorduz** [comment]
@louismagowan take a look at https://github.com/pymc-labs/pymc-marketing/pull/916 ;) We show that the methods are equivalent. Moreover, the liftest through likelihoods allow us for more information to be passed to the model ;)
